go.mod

```diff
@@ -19,6 +19,8 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/anchore/enterprise-client-go v1.0.0 // indirect
```

Comment:
This is quite old, please use: github.com/anchore/enterprise-client-go v1.0.1-0.20230530214028-c5364e91c3b3 instead as that is the latest version